python/vtool/unreal_lib/usd.py

```diff
@@ -67,6 +67,7 @@ def import_file(filepath):
         if unreal_lib.util.is_control_rig(package_name):
             found_control_rig = package_name
 
+    mesh = None
```

Comment:
Isn't this default value approach you mention in the description?  I'm adding mesh = None in case `if found_skeletal_mesh:` doesn't pass